compiler/rustc_transmute_types/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
I don't think we really need to put this into a separate crate. Most type definitions like this just live in rustc_middle.